src/utils/contentful.ts

```diff
@@ -1,9 +1,11 @@
 import { createClient } from "contentful";
+import type { Entry } from "contentful-management/dist/typings/export-types";
 
-import type {
+import {
```

Comment:
```suggestion
import type {
```

Enélkül failel a build.